Makefile

```diff
@@ -18,12 +18,12 @@ all: lint test
 .PHONY: lint
 lint:
 	@echo "Running golangci-lint..."
-	@$(GO) tool golangci-lint run ./...
+	@golangci-lint run ./...
```

Comment:
This would use the local version rather than the one installed with go mod, did you have an issue here?